bin/cron/deploy_to_test

```diff
@@ -31,6 +31,31 @@ def main
 
   DevelopersTopic.set_dtt TOPIC_DTT_IN_PROGRESS
 
+  unless GitHub.compare(base: 'staging', head: 'test').empty?
```

Comment:
So, I wrote a helper for this yesterday and backed it out because I didn't find a compelling use case - but you've got one here.  The [compare API](https://developer.github.com/v3/repos/commits/#compare-two-commits) returns ahead/behind information which feels like a cleaner way to ask this question.

```
  "status": "behind",
  "ahead_by": 1,
  "behind_by": 2,
  "total_commits": 1,
```

So maybe `GitHub.ahead?(base: 'staging', head: 'test')`